integrations/io.js

```diff
@@ -97,6 +97,17 @@ module.exports = function ({
     },
     async removeFile(file) {
       let filePath = path.resolve(toolRoot, file)
+
+      // File doesn't exist, so we don't need to cache it.
+      if (
+        !(await fs
+          .access(filePath)
+          .then(() => true)
+          .catch(() => false))
+      ) {
+        return
+      }
+
       if (!fileCache[filePath]) {
         fileCache[filePath] = await fs.readFile(filePath, 'utf8')
```

Comment:
I think you can get rid of the .access.then.catch call if you do this:

```suggestion
        fileCache[filePath] = await fs.readFile(filePath, 'utf8').catch(() => null)
```

Reply:
Ah you'd need to check and early return still but might be worth it ¯\_(ツ)_/¯